src/test/resources/test.erl

```diff
@@ -0,0 +1,9 @@
+-module(test).
+-behaviour(application).
+-export([start/2, stop/1]).
+
+start(_Type, _Args) ->
+  io:fwrite("Hello world!~n", []). 
+
+stop(_) ->
+  ok.
```

Comment:
Btw, by being able to execute `erl` this comment becomes irrelevant, but you might want to know that this idea of using the `erlc test.erl` approach would have broken once the library is released because the file `test.erl` is inside the test resources folder, which disappears (as with all tests) when the artifact to be released gets packaged